src/mocks/http.php

```diff
@@ -35,6 +40,13 @@ class Http
 	protected static $_default_404_verbal = false;
 	protected static $_instance           = false;
 
+	/**
+	 * Constructor
+	 */
+	public function __construct() {
+		self::$_curl = new \WpOrg\Requests\Transport\Curl();
```

Comment:
Cannot use static for this as this is create per transport object.  You should not re-use same transport that can potentially collide if this class was to be created multiple instances.